### ISSUE TITLE
No data on topic /dense/dense_cloud

### ISSUE THREAD

**Issue by BhavyanshM** [CIFASIS/dense-sptam#4]
Although the program creates the .pcd files in the /root/.ros/clouds directory, there is no data on the ROS Topic "/dense/dense_cloud". Neither "rostopic echo /dense/dense_cloud" nor Rviz show any data on that topic.  Could you please help resolve this issue. Thank you !

**Comment by BhavyanshM** [comment]
@taihup Also how could we use an actual Stereo camera data for the input? I don't see any launch files or instructions that cater to that. Please help!

**Comment by taihup** [comment]
Please send me the output in terminal when you launch the dense-sptam. And all the steps that you are using to run the system. Also, send me the terminal output of the `rostopic list` command.

To use your own camera with dense-sptam please follow the issue [https://github.com/lrse/sptam/issues/16](https://github.com/lrse/sptam/issues/16)  to make S-PTAM works first, and then try to run in parallel the densification node. There several some others issues where people is trying to run S-PTAM by themself.

To do so, you should have knowledge on ROS.

**Comment by BhavyanshM** [comment]
So it turns out that Dense S-PTAM does generate the point cloud when I use the ROS Bag from your own portal "http://cifasis-conicet.gov.ar/taihu/datasets/KITTI/bags/". I earlier did not know that this portal existed so I tried to create a ROS Bag for the KITTI dataset myself using the "kitti2bag" ROS package and raw dataset from the KITTI website. The /dense/dense_cloud did not publish any data when using the ROS bag created using kitti2bag.

**Comment by BhavyanshM** [comment]
At the moment I am not much worried about getting the kitti2bag dataset working with Dense S-PTAM but I am more interested in getting an actual Stereo Camera working with Dense S-PTAM. So let me try out your instructions from Issue lrse/sptam#16 and get you the results.

**Comment by taihup** [comment]
Cool, I close this issue so. Feel free to open another issue if you have any problem.
